### ISSUE TITLE
Add ability to have separate download directories for Sonarr/Radarr

### ISSUE THREAD

**Issue by shardulbee** [wouterdebie/putioarr#7]
**Is your feature request related to a problem? Please describe.**
Both Sonarr and Radarr show the downloaded files of the other system in the activity page

**Describe the solution you'd like**
The transmission client allows setting a "category" in each of Sonarr and Radarr. But putioarr only configures a single downloads directory

**Describe alternatives you've considered**
I guess I could run separate instances of putioarr, but that seems wasteful

**Comment by wouterdebie** [comment]
Good point. Currently putioarr ignores any download-directory information from sonarr/radarr, but we might be able to use this to point to the right download directory. It seems that if you use a category, the category is added to the download directory, which we could use in putioarr.

The tricky part is that currently putioarr doesn't keep any state, but solely relies on state that put.io provides and the download directory is not communicated to put.io. I did this to keep synchronization issues and state management to a minimum but I think it's time to rethink how I do this, since #5 will also benefit from keeping actual state. 

I don't have much time in the foreseeable future, but I'll keep this open.

**Comment by acidRain-burns** [comment]
I find the TOML config pretty straight-forward.

Regarding the state, maybe it is possible to include something under each -arr settings:
```
### -arr Services
# Each of the following services are optional, but you'll need at least one of them.
# The following parameters are supported:
#
#        URL: The url of your -arr service instance. May be an IP address. Don't forget the port number.
#    API Key: Can be found in the service's web GUI under Settings -> General
#   Category: Can be set by you when setting up putioarr as transmission Download Client in the service.
#             Looks for a sub-directory with the same name in the default downloads folder.
#             Leave empty if you have not set one in the service for this Download Client.
#             THIS VALUE IS CASE-SENSITIVE. Please ensure it matches what you set in the service.
[sonarr]
url = "http://x.x.x.x:8989"
api_key = "x"
category = "shows"

[radarr]
url = "http://x.x.x.x:7878"
api_key = "x"
category = "movies"
```

Later, it might be nice to let each service optionally have its own directory set?

**Comment by monxas** [comment]
i'd love to have this feature, sine this is "strongly recommended" by sonarr and radarr themselves

**Comment by wouterdebie** [comment]
This has been implemented some time ago.. Please reopen if something is wrong.
